app/src/main/java/ch/epfl/sdp/blindly/profile_setup/ProfileGender.kt

```diff
@@ -26,15 +37,30 @@ class ProfileGender : AppCompatActivity() {
             }
             //more option is checked
             radioButtonMore.isChecked -> {
-                val intent = Intent(this, ProfileGenderMore::class.java)
-                startActivity(intent)
+                val intentMore = Intent(this, ProfileGenderMore::class.java)
+                val genre = "More"
```

Comment:
It would be better to create a private const val like in other classes. The same applies for the next 2 cases in the code.

Reply:
You mean, one private const val: man, woman and more or a var?

Reply:
It is probably clearer if they are separated in 3 distinct const vals.

Reply:
Okay! Again, I'll push it in the next branch (there's some refactoring that will touch the same class anyway)